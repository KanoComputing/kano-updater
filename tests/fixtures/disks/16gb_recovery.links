/dev/disks/by-label/boot -> ../../mmcblk0p6
/dev/disks/by-label/RECOVERY -> ../../mmcblk0p1
/dev/disks/by-label/root -> ../../mmcblk0p7
/dev/disks/by-label/SETTINGS -> ../../mmcblk0p5
