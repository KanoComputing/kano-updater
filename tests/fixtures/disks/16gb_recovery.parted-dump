Model: SD SS16G (sd/mmc)
Disk /dev/mmcblk0: 31116288s
Sector size (logical/physical): 512B/512B
Partition Table: msdos
Disk Flags: 

Number  Start     End        Size       Type      File system  Flags
 1      8192s     3386718s   3378527s   primary   fat32        lba
 2      3386719s  31116287s  27729569s  extended
 5      3391488s  3457021s   65534s     logical   ext4
 6      3457024s  3866621s   409598s    logical   fat16        lba
 7      3866624s  31116287s  27249664s  logical   ext4

