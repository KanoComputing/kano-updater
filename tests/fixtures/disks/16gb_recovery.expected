{
    "commands": [
        "parted unit s resizepart 2 31116287",
        "parted unit s resizepart 7 31116287",
        "partprobe /dev/mmcblk0",
        "resize2fs /dev/mmcblk0p7"
    ]
}
